spinalcordtoolbox/reports/qc2.py

```diff
@@ -538,16 +575,45 @@ def sct_deepseg_sagittal(
     ) if fname_seg_lesion else None
     if fname_seg_lesion:
         img_seg_lesion.data = (img_seg_lesion.data > 0.5) * 1
-
-    # The radius is set to display the entire slice
-    radius = (img_input.dim[1]//2, img_input.dim[2]//2)
+    img_qc_seg = resample_nib(
+        image=img_qc_seg,
+        image_dest=img_input,
+        interpolation='linear',
+    ) if fname_qc_seg else None
+
+    # Using -qc-seg mask if available, we remove slices which are empty (except for the first 3 and last 3 slices just around the segmented slices)
+    if fname_qc_seg:
+        # We dilate the mask to keep the first and last 2 slices
+        first_slice = min(np.where(img_qc_seg.data)[0]) - 2
+        last_slice = max(np.where(img_qc_seg.data)[0]) + 2
```

Comment:
The reason I suggest using a function is that it helps prevent discrepancies like this... (this one has `2` instead of `3`).

Reply:
I did that on purpose since because images are bigger on sagittal slices. Having one less slice occupies significantly less space.

Reply:
Ah, okay!! Thank you for the clarification. (The comment above still says "three" so I wasn't sure, hehe.)

Reply:
Modification done!